docs/sources/setup-grafana/installation/helm/index.md

```diff
@@ -0,0 +1,374 @@
+
+In order to make any configuration changes, download the `values.yaml` file from the Grafana Helm Charts repository:
+
+https://github.com/grafana/helm-charts/edit/main/charts/grafana/values.yaml
+
+{{% admonition type="note" %}}
+Depending on your use case requirements, you can use a single YAML file that contains your configuration changes or you can create multiple YAML files.
+{{% /admonition %}}
+
+### Enable persistent storage **(recommended)**
+
+By default, persistent storage is disabled, which means that Grafana uses ephemeral storage, and all data will be stored within the container's file system. This data will be lost if the container is stopped, restarted, or if the container crashes.
+
+It is highly recommended that you enable persistent storage in Grafana Helm charts if you want to ensure that your data persists and is not lost in case of container restarts or failures.
+
+Enabling persistent storage in Grafana Helm charts ensures a reliable solution for running Grafana in production environments.
+
+To enable the persistent storage in the Grafana Helm charts, complete the following steps:
+
+1. Open the `values.yaml` file in your favorite editor.
+
+2. Edit the values and under the section of `persistence`, change the `enable` flag from `false` to `true`
+
+   ```yaml
+   .......
+   ............
+   ......
+   persistence:
+   type: pvc
+   enabled: true
+   # storageClassName: default
+   .......
+   ............
+   ......
+   ```
+
+3. Run the following `helm upgrade` command by specifying the `values.yaml` file to make the changes take effect:
+
+   ```bash
+   helm upgrade my-grafana grafana/grafana -f values.yaml -n monitoring
+   ```
+
+After that, the PVC will be enabled and able to store all of your data e.g. Dashboards, Data sources, etc.
+
+### Install plugins (e.g. Zabbix app, Clock panel, etc.)
+
+You can install plugins in Grafana from the official and community [plugins page](https://grafana.com/grafana/plugins). These plugins allow you to add new visualization types, data sources, and applications to help you better visualize your data.
+
+Grafana currently supports three types of plugins: panel, data source, and app. For more information on managing plugins, refer to [Plugin Management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+To install plugins in the Grafana Helm Charts, complete the following steps:
+
+1. Open the `values.yaml` file in your favorite editor.
+
+2. Find the line that says `plugins:` and under that section, define the plugins that you would like to install e.g.
+
+   ```yaml
+   .......
+   ............
+   ......
+   plugins:
+   # here we are installing two plugins, make sure to keep the indentation correct as written here.
+
+   - alexanderzobnin-zabbix-app
+   - grafana-clock-panel
+   .......
+   ............
+   ......
+   ```
+
+3. After that save the changes and use the `helm upgrade` command to get these plugins installed:
```

Comment:
You do not need to ever use the words "After that" in a task because the numbered step implies the concept of "after".